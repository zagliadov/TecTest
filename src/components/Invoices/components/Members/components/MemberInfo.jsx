import React from "react";

export const MemberInfo = ({ user }) => {
  return (
    <div className="py-2 px-2 border-b-2">
      <p>{user.email}</p>
    </div>
  );
}
