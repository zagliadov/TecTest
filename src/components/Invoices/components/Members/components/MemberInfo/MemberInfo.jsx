import React from "react";

export const MemberInfo = ({ user = {} }) => {

  if(typeof user === 'undefined') return null;

  return (
    <div className="py-2 px-2 border-b-2">
      <p>{user.email}</p>
    </div>
  );
}
