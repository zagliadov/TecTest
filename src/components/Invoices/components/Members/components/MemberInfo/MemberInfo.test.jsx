import React from "react";
import { render, screen } from "@testing-library/react";
import { MemberInfo } from "./MemberInfo";

test("MemberInfo render", async () => {
  const { asFragment } = render(<MemberInfo />);
  const email = screen.getByText('hello');
  expect(email).toBeInTheDocument();
  expect(asFragment(<MemberInfo />)).toMatchSnapshot();
});
