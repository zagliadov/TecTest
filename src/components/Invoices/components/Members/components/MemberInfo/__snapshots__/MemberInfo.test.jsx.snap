// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MemberInfo render 1`] = `
<DocumentFragment>
  <div
    class="py-2 px-2 border-b-2"
  >
    <p />
    <p>
      hello
    </p>
  </div>
</DocumentFragment>
`;
